pkg/controller/migplan/pvlist.go

```diff
@@ -280,12 +280,14 @@ func (r *ReconcileMigPlan) getSupportedActions(pv core.PersistentVolume, claim m
 	if pv.Spec.NFS != nil ||
 		pv.Spec.AWSElasticBlockStore != nil {
 		return append(supportedActions,
+			migapi.PvSkipAction,
```

Comment:
These two additions only add `PvSkipAction` to cases where move is allowed -- i.e. EBS and NFS volumes, or in cases where mig-operator is configured to allow move for the storage class. NOte the default return below does not add skip. Also, skip is added above in cases where `!claim.HasReference`, so it may be added twice in some cases. I think the right thing to do here, assuming you always want to allow skip, is actually to simply remove the `if !claim.HasReference` check and always include skip.